src/index.js

```diff
@@ -2,6 +2,7 @@ const moment = require('moment');
 const crypto = require('crypto');
 const assign = require('lodash.assign');
 const request = require('request');
```

Comment:
Please remove request as well, also from the package.json! You removed this dependency with using https :)